### PR TITLE
lightning: docs change for 'disk-quota' GA

### PR DESCRIPTION
**Pull request by dsdashun**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Docs change to make Lightning's 'disk-quota' function GA.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-lightning/tidb-lightning-configuration.md [hunk 1]
```diff
@@ -296,6 +301,9 @@ analyze = "optional"
 switch-mode = "5m"
 # 在日志中打印导入进度的持续时间。
 log-progress = "5m"
+
+# 使用 Physical Import Mode 时，检查本地磁盘配额的时间间隔，默认为 60 秒。
+# check-disk-quota = "30s"
```
Comment (buchuitoudegou) [inline tidb-lightning/tidb-lightning-configuration.md:306]: 设置成默认值会比较好

**Review by buchuitoudegou — COMMENTED** [review]
rest lgtm

File: tidb-lightning/tidb-lightning-configuration.md (hunk 1 above)
Comment (dsdashun) [reply tidb-lightning/tidb-lightning-configuration.md]: fixed in 20f0608044ed9ed5e7aad29982bf0204ed79eb74

File: tidb-lightning/tidb-lightning-configuration.md [hunk 2]
```diff
@@ -139,6 +139,11 @@ addr = "172.16.31.10:8287"
 # Physical Import Mode 限制 TiDB Lightning 向每个 TiKV 节点写入的带宽大小，默认为 0，表示不限制。
 # store-write-bwlimit = "128MiB"
 
+# 使用 Physical Import Mode 时，配置 TiDB Lightning 本地临时文件使用的磁盘配额 (disk quota)。当磁盘配额不足时，TiDB Lightning 会暂停读取源数据以及写入临时文件的过程，优先将已经完成排序的 key-value 写入到 TiKV，TiDB Lightning 删除本地临时文件后，再继续导入过程。
+# 需要同时配合把 backend 设置为 "local" 模式才能生效。
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-configuration.md:143]:
```suggestion
# 需要同时配合把 `backend` 设置为 `local` 模式才能生效。
```

File: tidb-lightning/tidb-lightning-configuration.md [hunk 3]
```diff
@@ -296,6 +301,9 @@ analyze = "optional"
 switch-mode = "5m"
 # 在日志中打印导入进度的持续时间。
 log-progress = "5m"
+
+# 使用 Physical Import Mode 时，检查本地磁盘配额的时间间隔，默认为 60 秒。
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-configuration.md:305]: 取值范围是多少？

File: tidb-lightning/tidb-lightning-physical-import-mode-usage.md [hunk 4]
```diff
@@ -212,10 +212,6 @@ io-concurrency = 5
 
 ## 磁盘资源配额 <span class="version-mark">从 v6.2.0 版本开始引入</span>
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-physical-import-mode-usage.md:214]: 这个警告我已经在另外一个 PR 里删掉了 https://github.com/pingcap/docs-cn/pull/12090/files

**Review by buchuitoudegou — APPROVED** [review]

File: tidb-lightning/tidb-lightning-configuration.md (hunk 3 above)
Comment (buchuitoudegou) [reply tidb-lightning/tidb-lightning-configuration.md:305]: 大于 0。用户应该不会填负数吧。

File: tidb-lightning/tidb-lightning-configuration.md [hunk 5]
```diff
@@ -139,6 +139,11 @@ addr = "172.16.31.10:8287"
 # Physical Import Mode 限制 TiDB Lightning 向每个 TiKV 节点写入的带宽大小，默认为 0，表示不限制。
 # store-write-bwlimit = "128MiB"
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-configuration.md:141]: 下面新增内容同步到了英文 PR https://github.com/pingcap/docs/pull/11409/files

File: tidb-lightning/tidb-lightning-physical-import-mode-usage.md (hunk 4 above)
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-physical-import-mode-usage.md:214]:
```suggestion

> **警告：**
>
> 磁盘资源配额目前是实验性功能，不建议在生产环境中使用。

```

**Review by shichun-0415 — APPROVED** [review]

**Review by hfxsd — APPROVED** [review]

File: tidb-lightning/tidb-lightning-configuration.md [hunk 6]
```diff
@@ -139,6 +139,11 @@ addr = "172.16.31.10:8287"
...
 
+# 使用 Physical Import Mode 时，配置 TiDB Lightning 本地临时文件使用的磁盘配额 (disk quota)。当磁盘配额不足时，TiDB Lightning 会暂停读取源数据以及写入临时文件的过程，优先将已经完成排序的 key-value 写入到 TiKV，TiDB Lightning 删除本地临时文件后，再继续导入过程。
+# 需要同时配合把 `backend` 设置为 `local` 模式才能生效。
+# 默认值为 MaxInt64 字节，即 9223372036854775807 字节。
+# disk-quota = "10GB"
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-configuration.md:145]:
```suggestion
# disk-quota = "10GB" 
```

**Pull request merged**